app/main/spawn.js

```diff
@@ -23,8 +24,15 @@ const CONTENT_SERVICE_PATH = path.join(SERVICES_PATH, 'content-service', 'index.
 
 // @TODO Use something like `forever` or `pm2` to ensure that this
 // process remains running
-// @TODO Will `node` be accessible via path on users machines that don't
-// have node installed as an engineer?
+
+export function getElectronExecutable() {
```

Comment:
This should be smarter and use process.execPath when being run from electron already. Arguably we could always use process.execPath but I'd prefer to run from electron even when manually starting the services for consistent testing.